.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: python
+install: pip install flake8
+script: flake8 . --count --select=E9,F63,F72,F82 --show-source --statistics
```

Comment:
Maybe to be consistent with our other models we should do `flake8 . --max-line-length=127`?